src/UI/Implementation/Component/Symbol/Icon/Renderer.php

```diff
@@ -38,17 +38,9 @@ public function render(Component\Component $component, RendererInterface $defaul
             $tpl->setVariable("ABBREVIATION", $ab);
         }
 
-        $di = $component->isDisabled();
-        if ($di) {
-            $tpl->setVariable("DISABLED", " disabled");
-        }
-
-        $id = $this->bindJavaScript($component);
```

Comment:
Why did you remove this? I think we still need the ability to bind JS...

Reply:
When (re)reading the Icons's description, I found "Icons themselves are not interactive; however they are allowed  within interactive containers."
What kind of JS would you fathom?